Test suite failed to run, localstorage-related

Currently, I'm getting the following failures with our existing Jest Testing:

```bash
 FAIL  public/js/components/specs/number.spec.js  ● Test suite failed to run
    SecurityError: localStorage is not available for opaque origins
      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at Array.forEach (native)
 FAIL  public/js/lib/specs/colorContraster.spec.js  ● Test suite failed to run
    SecurityError: localStorage is not available for opaque origins
      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at Array.forEach (native)
 FAIL  public/js/lib/specs/guid.spec.js  ● Test suite failed to run
    SecurityError: localStorage is not available for opaque origins
      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at Array.forEach (native)
```

I'm not 100% sure what is causing it, but it seems big enough to launch into its own issue. Help for sure wanted, ping me for reproduction steps if you aren't seeing this!

Steps to reproduce, from the master branch:

`SIMULATING=true docker-compose up --build -d`

`docker container exec -it electric-io /bin/sh`

`npm run test`

**Console Output:**

```
> electric-io@0.1.0 test /usr/app
> jest

● Validation Warning:

  Unknown option "husky" with value {"hooks": {"pre-commit": "pretty-quick --staged"}} was found.
  This is probably a typing mistake. Fixing it will remove this message.

  Configuration Documentation:
  https://facebook.github.io/jest/docs/configuration.html

 FAIL  public/js/lib/specs/colorContraster.spec.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins

      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at Array.forEach (native)

 FAIL  public/js/components/specs/number.spec.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins

      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at Array.forEach (native)

 FAIL  public/js/lib/specs/guid.spec.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins

      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at Array.forEach (native)

-------------------------|----------|----------|----------|----------|-------------------|
File                     |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-------------------------|----------|----------|----------|----------|-------------------|
All files                |        0 |        0 |        0 |        0 |                   |
 lib                     |        0 |        0 |        0 |        0 |                   |
  helpers.js             |        0 |        0 |        0 |        0 |... 21,22,23,24,29 |
  liveHub.js             |        0 |      100 |        0 |        0 |             1,3,4 |
  simHub.js              |        0 |      100 |        0 |        0 |             1,3,4 |
  testSetup.js           |        0 |      100 |      100 |        0 |               1,2 |
 lib/simulator           |        0 |      100 |        0 |        0 |                   |
  runSimulator.js        |        0 |      100 |        0 |        0 |... 14,15,19,20,23 |
  simulatedDevice.js     |        0 |      100 |        0 |        0 |... 18,20,24,25,28 |
  simulatedDeviceList.js |        0 |      100 |      100 |        0 |            1,7,30 |
  simulatedHub.js        |        0 |      100 |        0 |        0 |... 20,21,24,25,28 |
  simulatedReceiver.js   |        0 |      100 |        0 |        0 |  1,2,5,8,10,11,14 |
 public/js               |        0 |      100 |        0 |        0 |                   |
  main.js                |        0 |      100 |        0 |        0 |           1,2,4,6 |
 public/js/lib           |        0 |        0 |        0 |        0 |                   |
  colorContraster.js     |        0 |        0 |        0 |        0 |... 42,43,48,49,50 |
  configuration.js       |        0 |      100 |        0 |        0 |  2,3,7,8,12,13,16 |
  guid.js                |        0 |        0 |        0 |        0 |       2,3,4,5,6,8 |
 public/js/utils         |        0 |      100 |      100 |        0 |                   |
  constants.js           |        0 |      100 |      100 |        0 |                 2 |
-------------------------|----------|----------|----------|----------|-------------------|
Test Suites: 3 failed, 3 total
Tests:       0 total
Snapshots:   0 total
Time:        3.966s
Ran all test suites.

npm ERR! Linux 4.9.93-linuxkit-aufs
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "test"
npm ERR! node v6.14.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! electric-io@0.1.0 test: `jest`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electric-io@0.1.0 test script 'jest'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electric-io package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     jest
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electric-io
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electric-io
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/app/npm-debug.log
```

**npm-debug.log contents:**

```
$ > cat npm-debug.log
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'test' ]
2 info using npm@3.10.10
3 info using node@v6.14.2
4 verbose run-script [ 'pretest', 'test', 'posttest' ]
5 info lifecycle electric-io@0.1.0~pretest: electric-io@0.1.0
6 silly lifecycle electric-io@0.1.0~pretest: no script for pretest, continuing
7 info lifecycle electric-io@0.1.0~test: electric-io@0.1.0
8 verbose lifecycle electric-io@0.1.0~test: unsafe-perm in lifecycle true
9 verbose lifecycle electric-io@0.1.0~test: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/usr/app/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
10 verbose lifecycle electric-io@0.1.0~test: CWD: /usr/app
11 silly lifecycle electric-io@0.1.0~test: Args: [ '-c', 'jest' ]
12 silly lifecycle electric-io@0.1.0~test: Returned: code: 1  signal: null
13 info lifecycle electric-io@0.1.0~test: Failed to exec test script
14 verbose stack Error: electric-io@0.1.0 test: `jest`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:920:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:230:5)
15 verbose pkgid electric-io@0.1.0
16 verbose cwd /usr/app
17 error Linux 4.9.93-linuxkit-aufs
18 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "test"
19 error node v6.14.2
20 error npm  v3.10.10
21 error code ELIFECYCLE
22 error electric-io@0.1.0 test: `jest`
22 error Exit status 1
23 error Failed at the electric-io@0.1.0 test script 'jest'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the electric-io package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     jest
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs electric-io
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls electric-io
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

--- a/package.json
+++ b/package.json
@@ -87,6 +87,7 @@
     "setupFiles": [
       "./lib/testSetup.js"
     ],
+    "testURL": "http://localhost/",
     "husky": {
       "hooks": {
         "pre-commit": "pretty-quick --staged"

